src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/item.scss

```diff
@@ -2,18 +2,34 @@
 
 $itemBackgroundColor: $alabaster;
 $itemTextColor: $blueZodiac;
+$itemTextActiveColor: $shakespeare;
 $itemTextColorDisabled: $silver;
+$itemActiveBackgroundColor: $wildSand;
 
 $itemBorder: 1px $wildSand solid;
 
+.container {
+    position: relative;
+    display: inline-block;
+
+    &.has-dropdown-open {
+        .item {
+            color: $itemTextActiveColor;
+            background-color: $itemActiveBackgroundColor;
+        }
+    }
+}
+
 .item {
+    position: relative;
+    z-index: 5;
```

Comment:
Try to use [stacking contexts](https://philipwalton.com/articles/what-no-one-told-you-about-z-index/) to avoid high z-indexes. `1` should probably be just enough here (resp. why is it needed at all? To avoid the box-shadow?